app/assets/v2/fonts/Muli/muli-v12-latin-600.svg

```diff
@@ -1,309 +0,0 @@
-<?xml version="1.0" standalone="no"?>
```

Comment:
Are these file deletions right for this branch?

Reply:
No I dont think so. not sure where they came through. should I just copy them from another branch and reimplement?

Reply:
@mbeacom file deletions undone, and named as in `Master`, my only question is on the migration files. Are those ok to add?